haproxy-ingress/templates/controller-template.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.controller.template -}}
+{{- if or .Values.controller.template .Values.controller.templateFile -}}
```

Comment:
missing this same logic in the `_podtemplate.yaml`, look for `controller.template`

Reply:
indeed, added